uber/config.py

```diff
@@ -646,6 +646,19 @@ def MENU_FILTERED_BY_ACCESS_LEVELS(self):
     def ADMIN_ACCESS_SET(self):
         return uber.models.AdminAccount.access_set()
 
+    @cached_property
```

Comment:
This can only ever be updated when we restart the server, so I figured it's safe to fully cache.